typeset/converter.js

```diff
@@ -0,0 +1,189 @@
+const path = require('path')
+const fs = require('fs')
+const pify = require('pify')
+const puppeteer = require('puppeteer')
+const writeFile = pify(fs.writeFile)
+require('dotenv').config()
+const bunyan = require('bunyan')
+let level = process.env.LOG_LEVEL || 'info'
+let log = bunyan.createLogger({
+  name: 'typeset converter',
+  streams: [
+    {
+      level: level,
+      stream: process.stdout
+    },
+    {
+      type: 'rotating-file',
```

Comment:
@PiotrKozlowski Is there a need for JS to handle log file rotation? I had thought that the python service would handle these. Also, it seems like creating log files in production code code could become problematic